src/dotnet-scaffolding/dotnet-scaffold-aspire/Program.cs

```diff
@@ -96,7 +96,7 @@
 //TODO separate adding transient steps from singleton services.
 static void ConfigureServices(IServiceCollection services)
 {
-    services.AddTransient<ValidateContextStep>();
+    services.AddTransient<ValidateOptionsStep>();
```

Comment:
One thing to consider if we do keep the ValidateOptionsStep concept, is:
1. Move it into Core
2. Register it in services as part of the core services so that users don't need to register it themselves to use it.

Reply:
ok added a TODO for now